src/main/java/frc/trigon/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,130 @@
+        driveLeftMotor(voltageMeasure.plus(Units.Volts.of(calculateStayingInPlaceVoltage(leftMotor.getSignal(TalonFXSignal.POSITION), targetState.affectedByRobotWeight))));
+    }
+
+    @Override
+    public void updateLog(SysIdRoutineLog log) {
+        log.motor("RightClimberMotor")
+                .linearPosition(Units.Meters.of(rightMotor.getSignal(TalonFXSignal.POSITION)))
+                .linearVelocity(Units.MetersPerSecond.of(rightMotor.getSignal(TalonFXSignal.VELOCITY)))
+                .voltage(Units.Volts.of(rightMotor.getSignal(TalonFXSignal.MOTOR_VOLTAGE) - calculateStayingInPlaceVoltage(rightMotor.getSignal(TalonFXSignal.POSITION), targetState.affectedByRobotWeight)));
+        log.motor("LeftClimberMotor")
+                .linearPosition(Units.Meters.of(leftMotor.getSignal(TalonFXSignal.POSITION)))
+                .linearVelocity(Units.MetersPerSecond.of(leftMotor.getSignal(TalonFXSignal.VELOCITY)))
+                .voltage(Units.Volts.of(leftMotor.getSignal(TalonFXSignal.MOTOR_VOLTAGE) - calculateStayingInPlaceVoltage(leftMotor.getSignal(TalonFXSignal.POSITION), targetState.affectedByRobotWeight)));
+    }
+
+    @Override
+    public void setBrake(boolean brake) {
+        rightMotor.setBrake(brake);
+        leftMotor.setBrake(brake);
+    }
+
+    @Override
+    public SysIdRoutine.Config getSysIdConfig() {
+        return ClimberConstants.SYSID_CONFIG;
+    }
+
+    @Override
+    public void stop() {
+        rightMotor.stopMotor();
+        leftMotor.stopMotor();
+    }
+
+    public boolean atTargetState() {
+        return Math.abs(toMeters(rightMotor.getSignal(TalonFXSignal.POSITION)) - targetState.positionMeters) < ClimberConstants.CLIMBER_TOLERANCE_METERS &&
+                Math.abs(toMeters(leftMotor.getSignal(TalonFXSignal.POSITION)) - targetState.positionMeters) < ClimberConstants.CLIMBER_TOLERANCE_METERS;
+    }
+
+    void driveRightMotor(Measure<Voltage> voltageMeasure) {
+        rightMotor.setControl(voltageRequest.withOutput(voltageMeasure.in(Units.Volts)));
+    }
+
+    void driveLeftMotor(Measure<Voltage> voltageMeasure) {
+        leftMotor.setControl(voltageRequest.withOutput(voltageMeasure.in(Units.Volts)));
+    }
+
+    void setTargetState(ClimberConstants.ClimberState targetState) {
+        this.targetState = targetState;
+        setTargetPosition(targetState.positionMeters, targetState.positionMeters, targetState.affectedByRobotWeight);
+    }
+
+    void setTargetPosition(double targetRightPositionMeters, double targetLeftPositionMeters, boolean affectedByRobotWeight) {
+        rightMotor.setControl(determineRequest(affectedByRobotWeight)
+                .withPosition(targetRightPositionMeters)
+                .withFeedForward(calculateStayingInPlaceVoltage(rightMotor.getSignal(TalonFXSignal.POSITION), affectedByRobotWeight)));
```

Comment:
The calculateStayingInPlaceVoltage indicates voltage and it's being applied to a feedforward?